### PR TITLE
VA-1785 DRM models

### PR DESCRIPTION
**Pull request by kevinzetterstrom**
#### Ticket
[VA-1785](https://vimean.atlassian.net/browse/VA-1785)

#### Ticket Summary
Create models for the DRM representations

#### Implementation Summary
Created a ```Drm``` class, and updated a few other classes for their updated representations.

#### How to Test
Can use the drm-poc branch of Vimeo-Android to test the models.

### CODE REVIEW

**Comment by kevinzetterstrom** [comment]
Bump @alfiehanssen

File: vimeo-networking/src/main/java/com/vimeo/networking/model/playback/Drm.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import com.vimeo.stag.GsonAdapterKey;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+
+/**
+ * An object that holds on to the drm content. There are three types, fairplay, widevine, and playready.
+ * Since this is a Java library and only Apple products support fairplay, that type is omitted. Clients will
+ * only receive these if given the app-specific permission.
```
Comment (alfiehanssen) [inline vimeo-networking/src/main/java/com/vimeo/networking/model/playback/Drm.java]: Might be worth being more explicit here in the comment, noting that this is essentially a private class.

File: vimeo-networking/src/main/java/com/vimeo/networking/model/playback/VideoLog.java [hunk 2]
```diff
@@ -44,6 +46,11 @@
     @GsonAdapterKey("play_link")
     public String mPlayLink;
 
+    // URL for the "exit" event, used for drm purposes
```
Comment (alfiehanssen) [inline vimeo-networking/src/main/java/com/vimeo/networking/model/playback/VideoLog.java]: Augment comment to describe how it comes into play?

File: vimeo-networking/src/main/java/com/vimeo/networking/model/playback/VideoLog.java [hunk 3]
```diff
@@ -68,6 +75,15 @@ public String getPlayLoggingUrl() {
         return mPlayLink;
     }
 
+    public boolean isExitLinkEmpty() {
```
Comment (alfiehanssen) [inline vimeo-networking/src/main/java/com/vimeo/networking/model/playback/VideoLog.java]: Opportunity to just have a string method called `isEmpty`so this same logic isn't duplicated for each string (null check and isEmpty check)?

**Comment by alfiehanssen** [comment]
👍 Looks great @kevinzetterstrom 🎉

**Comment by kevinzetterstrom** [comment]
Good suggestions, added here https://github.com/vimeo/vimeo-networking-java/pull/104/commits/573c814c2c9df28b49dce893466aa5a66626478e

**Pull request merged**